Wrappers/Python/cil/optimisation/algorithms/SIRT.py

```diff
@@ -102,29 +102,30 @@ def set_up(self, initial, operator, data, lower=None, upper=None, constraint=Non
         logging.info("{} setting up".format(self.__class__.__name__, ))
         
         self.x = initial.copy()
+        self.tmp_x = self.x * 0.0
         self.operator = operator
         self.data = data
         
         self.r = data.copy()
         
-        self.relax_par = 1.0
+        # relaxation parameter was set to 1.0 and not exposed to the user: removed
```

Comment:
Let us expose it instead.

Reply:
If leaving this for another issue, please change comment, as is not meaningful as is referring to some previous functionality changed by this commit.

Reply:
relaxation parameter is exposed now